contracts/solidity/oz/safeCast/SafeCast.sol

```diff
@@ -0,0 +1,328 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.20;
+
+import "@openzeppelin/contracts/utils/math/SafeCast.sol";
+
+contract SafeCastTest {
+    uint256 maxUint256 = type(uint256).max;
+    uint248 maxUint248 = type(uint248).max;
+    uint240 maxUint240 = type(uint240).max;
+    uint232 maxUint232 = type(uint232).max;
+    uint224 maxUint224 = type(uint224).max;
+    uint216 maxUint216 = type(uint216).max;
+    uint208 maxUint208 = type(uint208).max;
+    uint200 maxUint200 = type(uint200).max;
+    uint192 maxUint192 = type(uint192).max;
+    uint184 maxUint184 = type(uint184).max;
+    uint176 maxUint176 = type(uint176).max;
+    uint168 maxUint168 = type(uint168).max;
+    uint160 maxUint160 = type(uint160).max;
+    uint152 maxUint152 = type(uint152).max;
+    uint144 maxUint144 = type(uint144).max;
+    uint136 maxUint136 = type(uint136).max;
+    uint128 maxUint128 = type(uint128).max;
+    uint120 maxUint120 = type(uint120).max;
+    uint112 maxUint112 = type(uint112).max;
+    uint104 maxUint104 = type(uint104).max;
+    uint96 maxUint96 = type(uint96).max;
+    uint88 maxUint88 = type(uint88).max;
+    uint80 maxUint80 = type(uint80).max;
+    uint72 maxUint72 = type(uint72).max;
+    uint64 maxUint64 = type(uint64).max;
+    uint56 maxUint56 = type(uint56).max;
+    uint48 maxUint48 = type(uint48).max;
+    uint40 maxUint40 = type(uint40).max;
+    uint32 maxUint32 = type(uint32).max;
+    uint24 maxUint24 = type(uint24).max;
+    uint16 maxUint16 = type(uint16).max;
+    uint8 maxUint8 = type(uint8).max;
+
+    int256 maxInt256 = type(int256).max;
+    int248 maxInt248 = type(int248).max;
+    int240 maxInt240 = type(int240).max;
+    int232 maxInt232 = type(int232).max;
+    int224 maxInt224 = type(int224).max;
+    int216 maxInt216 = type(int216).max;
+    int208 maxInt208 = type(int208).max;
+    int200 maxInt200 = type(int200).max;
+    int192 maxInt192 = type(int192).max;
+    int184 maxInt184 = type(int184).max;
+    int176 maxInt176 = type(int176).max;
+    int168 maxInt168 = type(int168).max;
+    int160 maxInt160 = type(int160).max;
+    int152 maxInt152 = type(int152).max;
+    int144 maxInt144 = type(int144).max;
+    int136 maxInt136 = type(int136).max;
+    int128 maxInt128 = type(int128).max;
+    int120 maxInt120 = type(int120).max;
+    int112 maxInt112 = type(int112).max;
+    int104 maxInt104 = type(int104).max;
+    int96 maxInt96 = type(int96).max;
+    int88 maxInt88 = type(int88).max;
+    int80 maxInt80 = type(int80).max;
+    int72 maxInt72 = type(int72).max;
+    int64 maxInt64 = type(int64).max;
+    int56 maxInt56 = type(int56).max;
+    int48 maxInt48 = type(int48).max;
+    int40 maxInt40 = type(int40).max;
+    int32 maxInt32 = type(int32).max;
+    int24 maxInt24 = type(int24).max;
+    int16 maxInt16 = type(int16).max;
+    int8 maxInt8 = type(int8).max;
+
+    function testToUint256(int256 number) public pure returns (uint256) {
```

Comment:
Didn't have a look in full but these functions shouldn't have the word "test" at the beginning. Please take a look at this ticket for more information https://github.com/hashgraph/hedera-smart-contracts/issues/578.

Reply:
fixed